LICENSE.md

```diff
@@ -0,0 +1,286 @@
+# Event Store License
```

Comment:
Review dependencies... check if copyright needs to be LTD, check dates. get JN to review